test/integration/integration.test.js

```diff
@@ -0,0 +1,41 @@
+import { exec } from "node:child_process"
+import { copyFileSync, rmSync } from 'node:fs'
+
+const meta_bench = "meta: 0x0000000000000000000000000000000000000000000000000000000000000001"
+const data_bench = "data: 0xf151b2c51f0885684a502d9e901846d9ffce3d4a000000000000000000000000"
+
+const rmdb =()=> {
+    rmSync("test/integration/fail_config/locktopus.sqlite", {force: true})
+    rmSync("test/integration/pass_config/locktopus.sqlite", {force: true})
+}
+
+describe('use real dmap, set your eth RPC url at test/integration/pass_config/config.jams', () => {
```

Comment:
this test should probably either be changed or be squelched / show warn when there is no config, tests should run in a clean environment with just the npm devdeps

Reply:
This one isn't included in "npm run test" for that reason, it will run with "npm run test-integration" as an optional extra to add testing beyond what tests with mocks won't reveal